product-web-app/model/addJobRole.ts

```diff
@@ -0,0 +1,5 @@
+export interface AddJobRole {
+  title?: string;
+  summary?: string;
+  sharepoint_link?: string;
```

Comment:
https://github.com/airbnb/javascript#naming-conventions
```suggestion
  sharepointLink?: string;
```